codecs/VWR.js

```diff
@@ -0,0 +1,74 @@
+/*
+* RPM codec
```

Comment:
These comments don't really serve a purpose I think and can be omitted, as they are frequently forgotten.